Makefile

```diff
@@ -152,3 +165,14 @@ endif
 
 	cat /tmp/cassandra-stress-$(STRESS_TYPE).yaml
 	kubectl apply -f /tmp/cassandra-stress-$(STRESS_TYPE).yaml
+
+# Generate bundle manifests and metadata, then validate generated files.
+bundle: generate
+	operator-sdk generate kustomize manifests -q
+	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	operator-sdk bundle validate ./bundle
+
+# Build the bundle image.
+bundle-build:
+	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
```

Comment:
What's this ? do we need it ?

Reply:
For running `operator-sdk bundle validate` it is not required, so I can drop it. But this is part of bundle support that operator-sdk suggests.